### DISCUSSION TITLE
How to get a signed `tx_blob` from a given transaction `struct` instance.

### DISCUSSION THREAD

**Discussion by asteshin** [sephynox/xrpl-rust#65] [General]
Hi everyone. It's not clear for me now, so I would like to ask you for a quite simple example of serialization and signing.

**Comment by LimpidCrypto** [comment]
Hi, we don't have a high level function to sign a transaction yet. We currently have it planned to be added in June. However, I see the importance of this feature. We may be able to add support for signing earlier. It depends on how fast we can implement the clients and the following models in May:
- Models for each API method response
- Response model
- Metadata Models

**Reply by asteshin** [reply]
I see. A question about binary serialization. How can I do it using a `BinarySerializer`? So that I will try to implement it for a first time.

**Comment by LimpidCrypto** [comment]
@asteshin keep an eye on #71
